### PR TITLE
Foldered service docs

### PR DESCRIPTION
**Pull request by sascala**

### CODE REVIEW

File: frameworks/kafka/docs/install-and-customize.md [hunk 1]
```diff
@@ -96,6 +96,41 @@ Installing multiple Kafka clusters is identical to installing Kafka clusters wit
 
     $ dcos package install kafka --options=kafka1.json
 
+		<!-- THIS BLOCK DUPLICATES THE OPERATIONS GUIDE -->
+
+## Integration with DC/OS access controls
+
+In Enterprise DC/OS 1.10 and above, you can integrate your SDK-based service with DC/OS ACLs to grant users and groups access to only certain services. You do this by installing your service into a folder, and then restricting access to some number of folders. Folders also allow you to namespace services. For instance, `staging/kafka` and `production/kafka`.
```
Comment (nickbp) [inline frameworks/kafka/docs/install-and-customize.md]: IIRC folder support isn't specific to enterprise. However it does require 1.10+.

Comment (sascala) [reply frameworks/kafka/docs/install-and-customize.md]: Ahh OK. Will test.

Comment (sascala) [reply frameworks/kafka/docs/install-and-customize.md]: Tested. Unless there's something I'm not seeing, you can't create ACLs in OSS.

Comment (nickbp) [reply frameworks/kafka/docs/install-and-customize.md]: Did the installation of the service into a folder itself work? I guess the phrasing here is specifically about ACLs

**Comment by nickbp** [comment]
That was the only comment I had, and strictly speaking the doc sounds accurate

**Pull request merged**